core/hoverfly_funcs.go

```diff
@@ -40,8 +42,8 @@ func (hf *Hoverfly) DoRequest(request *http.Request) (*http.Response, error) {
 		resp.Header.Add("Hoverfly", "Forwarded")
 	}
 
-	return resp, nil
-
+	elapsed := time.Since(start)
```

Comment:
I think the intention is to measure the elapsed time of the outbound call, so moving this to L34 would be more accurate.

Reply:
done